.github/dockerfiles/ov_build/ubuntu_20_04_arm64/Dockerfile

```diff
@@ -47,10 +44,6 @@ RUN chmod +x /install_build_dependencies.sh && \
     /install_build_dependencies.sh && \
     rm -rf /var/lib/apt/lists/*
 
-# Set gcc-10 as a default compiler
-RUN update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-10 30 && \
-    update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-10 30
```

Comment:
for arm64 gcc-10 is required

you can add a note that "it's required for multi-isa build"

Reply:
The only thing that did fail on Linux ARM64 with gcc-9 was the onnxruntime job, I made a w/a: https://github.com/openvinotoolkit/openvino/pull/26993/files#diff-65406737b8fad6efcaffc64bea9c6046909ed1e62620cb82769455b12cad1c5cR73. Other tests passed: https://github.com/openvinotoolkit/openvino/actions/runs/11288923919/job/31397744179.

Or do you mean that gcc-10 is a requirement to work with OpenVINO on Ubuntu 20?

iirc, Linux ARM64 is built using gcc-9 in Jenkins.